Tasks/XamarinLicense/XamarinLicense.ps1

```diff
@@ -12,7 +12,7 @@ Write-Verbose "email = $email"
 Write-Verbose "activateAndroid = $activateAndroid"
 Write-Verbose "timeout = $timeout"
 
-$activateAndroidLicense = Convert-String $activateAndroid Boolean
+$activateAndroidLicense = [System.Convert]::ToBoolean($activateAndroid)
```

Comment:
I merged the other PR after reviewing offline with Yang. It should fix your issue. You can close this PR now if so. Thanks for identifying this issue!